src/include/parser/query/reading_clause/load_from.h

```diff
@@ -0,0 +1,24 @@
+#pragma once
+
+#include "parser/expression/parsed_expression.h"
+#include "reading_clause.h"
+
+namespace kuzu {
+namespace parser {
+
+class LoadFrom : public ReadingClause {
+public:
+    LoadFrom(std::vector<std::string> filePaths, parsing_option_t parsingOptions)
+        : ReadingClause{common::ClauseType::LOAD_FROM}, filePaths{std::move(filePaths)},
+          parsingOptions{std::move(parsingOptions)} {}
+
+    inline std::vector<std::string> getFilePaths() const { return filePaths; }
+    inline const parsing_option_t& getParsingOptionsRef() const { return parsingOptions; }
```

Comment:
Should we return a raw pointer in this case? To avoid: auto option = getParsingOptionsRef();